include/inviwo/core/util/glm.h

```diff
@@ -812,6 +812,11 @@ struct almostEqual<glm::mat<C, R, T, Q>> {
 
 }  // namespace detail
 
+template <typename M, typename T = M::value_type>
+M filledMat(T v) {
```

Comment:
Would it work with just fill? Or filled?

Reply:
I'm fine with either. My though with including the Mat part is to indicate that the function is aimed for matrices (since it is not needed for vectors), ´util::filled<T>(v)´ could look like it also works on other datatypes as well (std::vector, images etc).

Reply:
I guess, technically, the function works for any datatype that has a single value constructor and operator+ taking a M::value_type parameters.

Reply:
for generic code using both vec and mat, it can be nice I think... might add a comment about intended usage?

Reply:
I would call it just filled